### DISCUSSION TITLE
<title>just me want my badge2

### DISCUSSION THREAD

**Discussion by EROR4** [community/community#114640] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

just me want my badge2

**Comment by h0murasak1** [comment]
horey

**Reply by h0murasak1** [reply]
well

**Comment by amardotai** [comment]
Then accept my answer so I could get my badge too.

**Reply by EROR4** [reply]
i can't, just make your own q&a

**Reply by queenofcorgis** [reply, marked as answer]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
